gslib/utils/stet_util.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for Split Trust Encryption Tool integration."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import os
+import shutil
+
+from gslib import storage_url
+from gslib.utils import execution_util
+from gslib.utils import temporary_file_util
+
+from boto import config
+
+DEFAULT_STET_CONFIG_PATH = '~/.config/stet.yaml'
```

Comment:
will the user know about creating the path at this location? Is this a common place for stet yaml? If not then maybe we should just avoid the default and let the user explicitly mention this path in the boto config? I think usually hidden files are written out by some tool. If the stet tool or gsutil is not doing this for the user, we should not expect user to write out a hidden file.

Reply:
This is the default path the STET team gave me. Maybe their binary or the install instructions create it.